app/views/scenarios/_form.html.erb

```diff
@@ -50,6 +50,19 @@
     <div class="col-md-4">
       <div class="form-group">
         <div>
+          <%= f.label :icon %>
+          <%= f.select(:icon, options_for_select([["Adjust", "fa fa-adjust", {"data-icon" => "fa-adjust"}],
+              ["Camera", "fa fa-camera", {"data-icon" => "fa-camera" }], ["Anchor", "fa fa-anchor", {"data-icon" => "fa-anchor" }], ["Archive", "fa fa-archive", {'data-icon'=> "fa-archive" }], ["Asterisk", "fa fa-asterisk", {'data-icon' => "fa-asterisk" }], ["Adn", "fa fa-adn", {'data-icon' => "fa-adn" }],["Bell", "fa fa-bell", {'data-icon' => "fa-bell" }], ["Bicycle", "fa fa-bicycle", {'data-icon' => "fa-bicycle" }]]), {},
+              {:style => "font-family:'FontAwesome', Arial;", :class => 'js-example-basic-single'}) %>
+        </div>
```

Comment:
Should we make a YAML file that lists the valid icons, or find some other way to get the list from the Gem?